hatch/src/repo.rs

```diff
@@ -39,6 +39,9 @@ pub fn clone_project_deps(path: &Path,
 {
   let mut visited: HashSet<String> = HashSet::new();
 
+  // All dependencies are cloned into here
+  let registry = &path;
```

Comment:
It took me a second to realize what this is, but I now think registry is a fine choice.